specification/data-faas.md

```diff
@@ -0,0 +1,29 @@
+# Semantic conventions for FaaS spans
+
+This document defines how to describe an instance of a function that runs without provisioning or managing of servers (also known as serverless) with spans.
+
+Span `name` should be set to the function name being executed. Depending on the trigger of the function, additional attributes MUST be set. For example, an HTTP trigger MUST follow the [HTTP Server semantic conventions](data-http.md#http-server-semantic-conventions).
```

Comment:
please avoid that long lines.

Reply:
I actually prefer long markdown lines. It means my editor can deal with line breaks how I have it configured, and the `suggestion` feature of github works much better if you only have to change a single line.